further_link/util/bluetooth/messages.py

```diff
@@ -0,0 +1,110 @@
+from typing import Optional
+
+
+class ChunkedMessageFormat:
+    START = b"START-"
+    STOP = b"STOP-"
```

Comment:
it would be nice to send data with the start and stop messages - maybe if they were 4 bytes long this would be made easier - the first 4 bytes are always a header and the reset is the data